.github/workflows/validate-action-dist.yml

```
@@ -0,0 +1,44 @@
+name: Validate Auto Assign Reviewer Action Build
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review, reopen]
```

actionlint (1.7.4)

invalid activity type "reopen" for "pull_request" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "edited", "enqueued", "labeled", "locked", "milestoned", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)